package.json

```diff
@@ -115,6 +114,27 @@
     "nexpect": "^0.4.2"
   },
   "optionalDependencies": {
-    "sl-blip": "http://blip.strongloop.com/strongloop@3.0.6"
+    "sl-blip": "http://blip.strongloop.com/strongloop@3.0.6",
+    "acorn": "^0.9.0",
+    "ansi-regex": "^1.1.1",
+    "bluebird": "^2.9.24",
+    "brace-expansion": "^1.1.0",
+    "core-util-is": "^1.0.1",
+    "inflight": "^1.0.4",
+    "inherits": "^2.0.1",
+    "lodash": "^2.4.1",
```

Comment:
After loopback-workspace has a release with strongloop/loopback-workspace#196 in it and loopback-boot has a 1.x release with strongloop/loopback-boot#128 in it, this lodash dependency should be switched to ^3.0.0 since the balance will be shifted so that 3.x has more duplicates.